### ISSUE TITLE
Three times learning rate decay.

### ISSUE THREAD

**Issue by PenFun** [WangYueFt/rfs#29]
Thank you very much for your work. I want to know which epochs after performing three times learning rate decays on the tieredImageNet, CIFAR-FS and FC100 datasets?

**Comment by z1358** [comment]
I am also interested in knowing at which epochs the learning rate is reduced, I think these would be very helpful for the reproduction of the results.
Or is it possible to provide pre-training or final models on these datasets?
Thanks!

**Comment by YiteWang** [comment]
Check here: https://github.com/WangYueFt/rfs/issues/5

**Comment by brando90** [comment]
@z1358 @YiteWang if you guys reproduce it let me know. I've not been able to.

See: https://github.com/WangYueFt/rfs/issues/34
